pkg/riff/commands/handler_invoke.go

```diff
@@ -82,10 +82,14 @@ func NewHandlerInvokeCommand(c *cli.Config) *cobra.Command {
 	opts := &HandlerInvokeOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "invoke",
-		Hidden:  true,
-		Short:   "<todo>",
-		Example: "<todo>",
+		Use:    "invoke",
+		Hidden: true,
```

Comment:
Why is this hidden?

Reply:
we're not sure if there should be an invoke command